scripts/update_tags_per_languages.py

```diff
@@ -0,0 +1,365 @@
+        file = os.path.abspath(os.path.join(os.path.dirname( __file__ ), '..', f'taxonomies/{plural}.txt')) 
+        
+        # country is needed otherwise <plural>_lc will be "en"
+        post_call_url = f"https://{user}{{country}}.openfoodfacts.{env}/cgi/product_jqm2.pl"
+
+
+        # 1) get all tags
+        all_tags = get_from_api(tags_list_url)
+        # example:
+        #  all_tags = {
+        #      "tags": [
+        #          {"id": "it:frankreich", "known": 0, "name": "frankreich", ...}, 
+        #      ],
+        #  }
+
+        # 2) fetch unknown tags and look into taxonomy
+        possible_wrong_language_tags = unknown_tags_taxonomy_comparison(all_tags, file, plural)
+
+        # limit number of iterations
+        # for dev, number of elements in possible_wrong_language_tags 
+        #   can be changed in unknown_tags_taxonomy_comparison()
+        for current_tag, updated_tag in possible_wrong_language_tags.items():
+
+            # 3) get all products for this tag 
+            all_products_for_tag = get_from_api(products_list_for_tag_url.format(tag_id_placeholder=current_tag))
+            # example:
+            #  all_products_for_tag = {
+            #      "products": [
+            #          {"categories": "Lait", "categories_lc": "en", ...}, 
+            #      ],
+            #  }
+
+            if dev:
+                i = 0
+            for product in all_products_for_tag["products"]:
+                if dev:
+                    if i > 0:
+                        break
+                    i += 1
+
+                # 4) update tags_fields 
+                updated_field = update_tags_field(product[plural], product[f'{plural}_lc'], current_tag, updated_tag)
+                
+                # 5) finally, update
+                if updated_field != product[plural] and not dev:
+
+                    # country is needed otherwise <plural>_lc will be "en"
+                    try:
+                        country = mapping_languages_countries[product[f'{plural}_lc']]
+                    except KeyError:
+                        print(f"ERROR: when updating product {product['code']}. Unknown country for this language: {product[f'{plural}_lc']}")
+                        sys.exit()
+
+                    data = {
+                        'code': product['code'], 
+                        plural: updated_field,
+                    }
+                    post_call_url_res = requests.post(
+                        post_call_url.format(country=country),
+                        data=data,
+                        headers=headers,
+                    )
+                    if post_call_url_res.status_code != 200:
+                        print(f"ERROR: when updating product {product['code']}. {post_call_url_res.status_code} status code")
+                        sys.exit()
+
+                with open(logs_file, "a") as f:
+                    f.write(f"{product['code']},{plural};{current_tag};{updated_tag}\n") 
```

Comment:
nit: you can also keep the file open, and use `f.flush()` to ensure you writes are immediately written on disk.

I find it great that you write what is processed. You could check for this file to avoid re-processing things already done in case the script failed at some point.